pkg/helm/chartproxy/testdata/mergedAzureRepoIndex.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: v1
```

Comment:
do we really need this file?

Reply:
adding this correct index file fixes the failing unit test